### ISSUE TITLE
Why the increase is not proportional?

### ISSUE THREAD

**Issue by gHashTag** [wix/react-native-autogrow-textinput#27]
Why the increase is not proportional?
![bug mov](https://user-images.githubusercontent.com/6774813/29495288-eff2d8a2-85c4-11e7-93c4-724f38026626.gif)

```
import React from 'react'
import PropTypes from 'prop-types'
import { AutoGrowingTextInput } from 'react-native-autogrow-textinput'

const AutoGrowingText = ({ value, onChangeText, placeholder }) => {
  const { textInput } = styles

  return (
    <AutoGrowingTextInput 
      value={value}
      style={textInput} 
      placeholder={placeholder} 
      onChangeText={onChangeText}
    />
  )
}

const styles = {
  textInput: {
    fontSize: 16,
    fontFamily: 'AppleSDGothicNeo-Light',
    paddingRight: 5,
    paddingLeft: 10,
    color: '#474747',
    backgroundColor: 'gold'
  },
}

AutoGrowingText.propTypes = {
  value: PropTypes.string,
  placeholder: PropTypes.string,
  onChangeText: PropTypes.func,
}

export { AutoGrowingText }
```

**Comment by AlanMoMo** [comment]
I encountered the same problem.

**Comment by webyak** [comment]
I'm experiencing the same issues. First thought it seemed to be something related to the font I'm using or the font-size but that doesn't seem to be the case.

**Comment by jordanmkoncz** [comment]
@gHashTag @AlanMoMo @webyak what version of `react-native` are you using? Versions `0.47.0` and `0.47.1` have a bug that appears to be the cause of this, see https://github.com/facebook/react-native/issues/15415. The bug seems to be fixed in `0.47.2` onwards.

**Comment by gHashTag** [comment]
@jordanmkoncz yes react-native: 0.48.3 worked! Thanks
